vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,203 @@
+    doi = "10.27861182.x"
+  )
+)
+covid_incubation
+```
+
+Not all arguments are specified in the example using the class constructor (`epidist()`) above, for example the `metadata` or parameter uncertainty (`uncertainty`) is not provided. See the help documentation for the `epidist()` function using `?epidist` to see each argument. Also see documentation for `<epidist>` helper functions, e.g., `?create_epidist_citation()`.
+
+Manually creating `<epidist>` objects can be especially useful if new parameter estimates become available but are not yet incorporated into the {epiparameter} library.
+
+As seen in the examples in this vignette, the `<epidist>` class has a custom printing method which shows the disease, pathogen (if known), the epidemiological distribution, a citation of the study the parameters are from and the probability distribution and parameter of that distribution (if available).
+
+::: {.alert .alert-success}
+### Benefit of `<epidist>`
+
+By providing a consistent and robust object to store epidemiological parameters, `<epidist>` objects can be applied in epidemiological pipelines, for example [{episoap}](https://github.com/epiverse-trace/episoap). The data contained within the object (e.g. parameter values, pathogen type, etc.) can be modified but the pipeline will continue to operate because the class is unchanged.
+:::
+
+The probability distribution (`prob_distribution`) argument requires the distribution specified in the standard R naming. In some cases these are the same as the distribution's name, e.g., `gamma` and `weibull`. Examples of where the distribution name and R name differ are lognormal and `lnorm`, negative binomial and `nbinom`, geometric and `geom`, and poisson and `pois`.
+
+## Subsetting database
+
+The database can be subset directly by `epidist_db()`. Here the results can be subset by author. It is recommended to use the family name of the first author instead of the full name. Only the first author will be matched when the entry is from a source with multiple authors.
+
+```{r epidist-db-subset-author}
+epidist_db(
+  disease = "COVID-19",
+  epi_dist = "incubation period",
+  author = "Linton"
+)
+```
+
+The results can be further subset using the `subset` argument, for example `subset = sample_size > 100` will return entries with a sample size greater than 100. See `?epidist_db()` for details on how to use this argument to subset which database entries get returned.
+
+## Adding library entries and contributing to {epiparameter}
+
+If a set of epidemiological parameter has been inferred and is known to the user but has not yet been incorporated into the {epiparameter} database, these parameters can be manually added to the library.
+
+```{r add-to-library}
+# wrap <epidist> in list to append to database
+new_db <- append(epi_dist_db, list(covid_incubation))
+```
+
+Note that this only adds the parameters to the library in the environment, and does not save to the database file in the package. 
```

Comment:
Here, would be useful to suggest what to do. Should they write the new db to disk?

```suggestion
Note that this only adds the parameters to the library in the environment, and does not save to the database file in the package. Hence, if you restart your R session, you will lose the changes. 
```

Reply:
On the point of writing to disk or contributing to the {epiparameter} database this has been raised in #304. I'll commit your improvement to the text and then work on how best a user can contribute or store an expanded database in the next version.